### ISSUE TITLE
Base URL without Rewrite Rules

### ISSUE THREAD

**Issue by Wulfheart** [flextype/flextype#291]
Some people don't have access to their server configuration. What about a variable named `baseurl` which provides the root url for the installation of flextype? 
This would be really helpful.

**Comment by Awilum** [comment]
@Wulfheart you may try this in twig templates 

`{{ base_url() }}`

**Comment by Wulfheart** [comment]
Unfortunately this isn't a possibility to view the admin panel, is it?

**Comment by Awilum** [comment]
@Wulfheart I don't understand what is a reason to change `base url` if it is already based on the sever where site is hosted.

**Comment by Wulfheart** [comment]
I'm running it in a subdirectory.

**Comment by Wulfheart** [comment]
To update this: I'm running it on a subdirectory and I don't have access to any RewriteRules.

**Comment by Wulfheart** [comment]
To clarify my issue:
I'm using flextype on shared hosting. There are several folders:
```
|
|---bin
|---html (here goes the actual content which will be displayed/invoked when the URL is called)
...
```
So flextypes URL is `my.custom.url/flextype`. Unfortunately with the links redirecting to `my.custom.url` as base it is impossible for me to use flextype. Moving the `html`-folder is no option as there are also other projects in this directory. Same applies to RewriteRules. They are not enabled by the provider.

By specifying a root url (in this example it would be `my.custom.url/flextype`) users would benefit in such use case with a trivial configuration option.

I hope I could clarify my issue properly. I'm looking forward to your opinions on this issue.

**Comment by Awilum** [comment]
implemented in 0.9.7 as twig function `{{ site_url() }}`  #373 
note: in 0.9.8 it will have another function name `{{ url() }}` #405
